character-set-and-collation.md

```diff
@@ -141,7 +141,7 @@ For details about the TiDB support of the GBK character set, see [GBK](/characte
 
 In MySQL, the character set `utf8` is limited to a maximum of three bytes. This is sufficient to store characters in the Basic Multilingual Plane (BMP), but not enough to store characters such as emojis. For this, it is recommended to use the character set `utf8mb4` instead.
 
-By default, TiDB provides the same 3-byte limit on `utf8` to ensure that data created in TiDB can still safely be restored in MySQL. This can be disabled by changing the value of `check-mb4-value-in-utf8` to `FALSE` in your TiDB configuration file.
+By default, TiDB provides the same 3-byte limit on `utf8` to ensure that data created in TiDB can still safely be restored in MySQL. This can be disabled by changing the value of the system variable `tidb_check_mb4_value_in_utf8` to `OFF`.
```

Comment:
```suggestion
By default, TiDB provides the same 3-byte limit on `utf8` to ensure that data created in TiDB can still safely be restored in MySQL. This can be disabled by changing the value of the system variable [`tidb_check_mb4_value_in_utf8`]((/system-variables.md#tidb_check_mb4_value_in_utf8) to `OFF`.
```